### ISSUE TITLE
Find a Store: No vocalization of results begin found

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13738]
**Describe the bug**
- If the user hits the "View All Stores" button, there is no verbal feedback that results have been found.
- If the user follows a country link, there's no verbal feedback that results are being displayed below or been found.

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**

1. Open any Screen Reader (VO, NVDA, JAWS).
2. Go to the Find a Store page
3. Click on the "View All Stores" button
4. Hear missing vocalization about results being found
5. Click on any location having stores
6. Hear missing vocalization about results being found

**Expected behavior**
- Screen reader users get vocalized feedback about results being found, best if detailing how many.
- Since this is a success/information message, use `aria-live="polite" aria-atomic="true"` to vocalize suggested messages below.

**Screenshots**

Message suggestion in the screenshot below, please validate phrasing with @Manurod2018 or @vekterdude

![Screen Shot 2020-11-24 at 6 18 24 PM](https://user-images.githubusercontent.com/15113219/133490558-bbc5af55-8da5-4f6a-90b0-d7d704bc3cff.png)

![Screen Shot 2020-11-24 at 6 33 20 PM](https://user-images.githubusercontent.com/15113219/133490575-d96afe3a-8a71-4c44-882d-ca494c290d6c.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus. 

![viewAllStores](https://user-images.githubusercontent.com/45409515/144297986-e60520c8-08ff-4386-b4c0-5b89be2ecc1f.png)

<img width="1313" alt="Screen Shot 2021-12-01 at 2 02 02 PM" src="https://user-images.githubusercontent.com/45409515/144297953-363584d4-d14b-475f-b70b-4c28d2960836.png">

<img width="1325" alt="Screen Shot 2021-12-01 at 2 05 34 PM" src="https://user-images.githubusercontent.com/45409515/144297956-78b3e964-5a22-4251-92df-077a1d366565.png">

More extensive testing (on other screenreaders and devices) will be done with RC.
